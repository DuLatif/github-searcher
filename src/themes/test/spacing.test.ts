import { describe, expect, test } from "vitest";
import { readFileSync } from "fs";

describe("Theme Spacing", () => {
  test("should match scss spacing variable snapshot", () => {
    expect(readFileSync(__dirname + "./../spacing.scss")).toMatchSnapshot();
  });
});
