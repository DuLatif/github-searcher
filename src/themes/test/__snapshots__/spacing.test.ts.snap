// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Spacing > should match scss spacing variable snapshot 1`] = `
{
  "data": [
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    48,
    58,
    32,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    58,
    32,
    52,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    50,
    58,
    32,
    56,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    51,
    58,
    32,
    49,
    50,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    52,
    58,
    32,
    49,
    54,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    53,
    58,
    32,
    50,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    54,
    58,
    32,
    50,
    52,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    55,
    58,
    32,
    50,
    56,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    56,
    58,
    32,
    51,
    50,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    57,
    58,
    32,
    51,
    54,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    48,
    58,
    32,
    52,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    49,
    58,
    32,
    52,
    56,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    50,
    58,
    32,
    54,
    52,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    51,
    58,
    32,
    56,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    52,
    58,
    32,
    57,
    54,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    112,
    97,
    99,
    101,
    45,
    49,
    53,
    58,
    32,
    49,
    50,
    56,
    112,
    120,
    59,
    13,
    10,
  ],
  "type": "Buffer",
}
`;
